src/components/SearchBar.vue

```diff
@@ -2,18 +2,18 @@
   <div>
     <a>{{ searchBoxDefault }}</a>
     <br />
-    <input v-model="searchBoxDefault" class="searchBar"  />
+    <input v-model="searchBoxDefault" class="searchBar" />
   </div>
 </template>
 
 <script>
 export default {
-    name: "SearchBar",
-    data: (() => {
-      return {
-        searchBoxDefault: "Begriff zum Suchen eingeben...",
-      }
-    },
+  name: "SearchBar",
+  data() {
```

Comment:
Since this is a prop of our object, there is a `:` missing after `data`.

Reply:
Changed